academy-of-management-review.csl

```diff
@@ -234,16 +252,23 @@
           <group>
             <text macro="container-contributors"/>
             <text macro="secondary-contributors"/>
-            <text variable="container-title" font-style="italic" font-weight="bold"/>
+            <choose>
+              <if type="article-journal article-magazine article-newspaper paper-conference" match="any">
```

Comment:
hmmm. And you'd actually expect something like Academy of Management Proceedings to appear in databases in sentence case? I'm fine testing for editor if you want to go that way, it just seems like a lot of complexity for few cases and I always worry that those types of rules may confuse end users. But as I said, if you want to put it in, I won't object. 
Please add a comment in the code explaining that, though, so later authors can make sense of it.